domain/miningmanager/mempool/handle_new_block_transactions.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/kaspanet/kaspad/domain/consensus/utils/transactionhelper"
 )
 
-func (mp *mempool) handleNewBlockTransactions(blockTransactions []*externalapi.DomainTransaction) (
+func (mp *mempool) handleNewBlockTransactions(blockTransactions []*externalapi.DomainTransaction, clone bool) (
```

Comment:
yes, that is useless, i will add the parameter into the function. I decided to clone everything out of the mempool because it seemed like it would be safer and good practice but my first implementation was with a parameter. But will keep it, just in case.

Reply:
yes, that is useless, i will change to cloning directly, where it is used, and add a comment why it is cloned.